src/ssh.c

```diff
@@ -935,3 +982,279 @@ const char* wolfSSH_GetSessionCommand(const WOLFSSH* ssh)
 
     return NULL;
 }
+
+
+int wolfSSH_worker(WOLFSSH* ssh, word32* channelId)
+{
+    int ret = WS_SUCCESS;
+
+    printf("Entering wolfSSH_worker()\n");
```

Comment:
this printf, and the rest in this function, should be mapped with macro for portability

Reply:
They need to be deleted. They were added for debugging when I didn't want the massive spray of all the other debugging messages.